obspy/io/nordic/core.py

```diff
@@ -34,10 +34,16 @@
     Tensor, ResourceIdentifier)
 
 
-mag_mapping = {"ML": "L", "MLv": "L", "mB": "B", "Ms": "s", "MS": "S",
+MAG_MAPPING = {"ML": "L", "MLv": "L", "mB": "B", "Ms": "s", "MS": "S",
                "MW": "W", "MbLg": "G", "Mc": "C"}
-
-onsets = {'I': 'impulsive', 'E': 'emergent'}
+INV_MAG_MAPPING = {item: key for key, item in MAG_MAPPING.items()}
+POLARITY_MAPPING = {"": "undecidable", "C": "positive", "D": "negative"}
+INV_POLARITY_MAPPING = {item: key for key, item in POLARITY_MAPPING.items()}
+ONSET_MAPPING = {'I': 'impulsive', 'E': 'emergent'}
+INV_ONSET_MAPPING = {item: key for key, item in ONSET_MAPPING.items()}
+EVALUATION_MAPPING = {'A': 'automatic', ' ': 'manual'}
+INV_EVALUTATION_MAPPING = {
+    item: key for key, item in EVALUATION_MAPPING.items()}
 # List of currently implemented line-endings, which in Nordic mark what format
 # info in that line will be.
 accepted_tags = ['1', '6', '7', 'E', ' ', 'F', 'M', '3']
```

Comment:
Mind capitalizing this for consistency? May also be worth making this a tuple just to make sure it doesn't mutate.

Reply:
I think I have actually repeated myself and should remove the non-caps version, thanks for catching it.  And good idea no the tuples :+1: